alerts_backend/python/app.py

```diff
@@ -1,14 +1,15 @@
 """Query alert information from AeroAPI and present it to a frontend service"""
 import os
-from datetime import timezone
+from datetime import *
```

Comment:
Best to avoid `*` imports so we always know where the symbols we use are coming from
```suggestion
from datetime import datetime
```